coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,15 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = '.hs',
+    versions = 1.0, 1.1, 1.2, 1.3, 1.4
+    special_chars = list('!#$%&*+./<=>?@\\^|-~')
+    comment_delimiters = '--',
+    multiline_comment_delimiters = {'{-': '-}'}
+    keywords = [
```

Comment:
Please refer to this [HaskellWiki](https://wiki.haskell.org/Keywords) for a complete list of Haskell keywords. There are several missing keywords. Also, why does underscore `_` listed as keyword but the other chars are not?

Reply:
Bump I still don't see all keywords listed at @wisn's URL here in the definition (for eaxmple `data instance`)

Reply:
@Makman2 `data` keyword and `instance` keyword already exists. I don't think we need to write it again as `data instance`. As for `data family`, since `family` is not a keyword by itself, I think we need to write it.